### DISCUSSION TITLE
Is it possible to pass property file value to testng.xml as parameter value?

### DISCUSSION THREAD

**Discussion by HamzaMiz** [appium/appium#16323] [Q&A]
Is it possible to pass property file value to testng.xml as parameter value?

Example:

test.properties:

browser = FIREFOX

testng.xml:

parameter name = "browser" value = "${test.browser}"

**Comment by jlipps** [comment]
This is not an Appium question. This seems like a TestNG question. Anyway, better off asking in the Appium forums or Selenium slack.
